ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -112,6 +112,90 @@ def test003_cancel_button_edit_departments(self, save):
                                                                                       order_departments))
             self.assertEqual(current_departments, order_departments)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test006_update_material_type(self, save):
+        """
+        New: Orders: Edit material type: Make sure that user able to change material type and related test plan &
+        article.
+
+        LIMS-4281
+        LIMS-4281
+        :return:
+        """
+        self.order_page.get_random_order()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        current_material_type = self.order_page.get_material_type()
+        self.order_page.set_material_type()
+        new_material_type = self.order_page.get_material_type()
+        self.order_page.confirm_popup(force=True)
+        self.order_page.get_article()
+        self.order_page.set_article()
```

Comment:
setting a general article may be lead to `empty` article.

Reply:
I did a solution get article only without set it ( I mean when i'm in suborders view I want to make sure that this article that I updated according to the materiel type it updated successfully in the suborder view )

Reply:
I already made it clear in set_test_plan() comment. So the solution is choosing a specific material type+active_article+test_plan to b valid